src/utilities/set-random-voice-channel-names.ts

```diff
@@ -0,0 +1,46 @@
+import type { Guild, VoiceBasedChannel } from 'discord.js';
+
+import { getRandomVoiceChannelNamesFromFile } from './get-random-voice-channel-names-from-file.js';
+
+/** Default channel name if a list is too short to foll all the slots */
+const DEFAULT_CHANNEL_NAME = 'Voice Channel';
+
+export async function setRandomVoiceChannelNames(guild: Guild): Promise<void> {
+  const { groups } = await getRandomVoiceChannelNamesFromFile();
+
+  // Get random group
+  const randomGroupIndex = Math.floor(Math.random() * groups.length);
+  const names = [...groups[randomGroupIndex].names];
+
+  // Get voice channels for the current server
+  const voiceChannels: readonly VoiceBasedChannel[] = Array.from(
+    guild.channels.cache
+  )
+    .map(([, channel]) => channel)
```

Comment:
This might just be weirdness with Prettier, but I don't particularly like how this is indented. Something about it makes the `.map` look like its floating/unrelated to the `.from` above it.

Reply:
Yep just prettier being weird. I broke it into another constant to make it a little better